src/Core/Repositories/IUserRepository.cs

```diff
@@ -13,12 +13,14 @@ public interface IUserRepository : IRepository<User, Guid>
     Task<User?> GetBySsoUserAsync(string externalId, Guid? organizationId);
     Task<UserKdfInformation?> GetKdfInformationByEmailAsync(string email);
     Task<ICollection<User>> SearchAsync(string email, int skip, int take);
+    Task<ICollection<UserDetails>> SearchDetailsAsync(string email, int skip, int take);
```

Comment:
Question: is this used somewhere? I might be missing it

Reply:
Its not! This was leftover from a previous implementation. I'll remove it.